src/components/templates/TemplateTerms.vue

```diff
@@ -1,408 +1,849 @@
+
+                    <div class="mb-3">
+                        <AtomParagraph
+                            :variation="'large'"
+                            id="cancellation-policies"
+                        >
+                            Cancellation Policies :
+                        </AtomParagraph>
+                        <ul class="pl-6" style="list-style-type: disc">
+                            <li>
+                                <AtomParagraph>
+                                    Cancellation request of the ongoing booking
+                                    must be made in one month advance.
+                                </AtomParagraph>
+                            </li>
+                        </ul>
+                    </div>
+
+                    <div class="mb-3">
+                        <AtomParagraph :variation="'large'" id="changes">
+                            Changes to These Terms and Conditions :
+                        </AtomParagraph>
+                        <ul class="pl-6" style="list-style-type: disc">
+                            <li>
+                                <AtomParagraph>
+                                    We reserve the right, at Our sole
+                                    discretion, to modify or replace these Terms
+                                    at any time.What constitutes a material
+                                    change will be determined at Our sole
+                                    discretion.By continuing to access or use
+                                    Our Service after those revisions become
+                                    effective, You agree to be bound by the
+                                    revised terms. If You do not agree to the
+                                    new terms, in whole or in part, please stop
+                                    using the website and the Service.
+                                </AtomParagraph>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+            </BodyWrapper>
+        </main>
+    </div>
 </template>
 
 <script>
-import BodyWrapper from '../extras/BodyWrapper.vue';
 import AtomHeading from '../atoms/AtomHeading.vue';
 import AtomParagraph from '../atoms/AtomParagraph.vue';
+import BodyWrapper from '../extras/BodyWrapper.vue';
+
 export default {
     name: 'TemplateTerms',
     components: {
-        BodyWrapper,
         AtomHeading,
         AtomParagraph,
+        BodyWrapper,
+    },
+    data() {
+        return {
+            activeSection: '',
+        };
     },
+    methods: {
+        scrollTo(id) {
+            const element = document.getElementById(id);
+            if (element) {
+              history.pushState(null, null, `#${id}`);
+              // Calculate offset based on screen size
+              const offset = window.innerWidth <= 768 ? -600 : window.innerWidth <= 1300 ? -10 : 10;
```

Comment:
its was better to have to variables as they added clarity what these condition mean.
If please not please add comment explaining the condition or you can split it into single methods
```
isMobile(width){
     return width < 768
}
```